test/simulator/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
 project(simulator CXX)
 
 # Build timestamp simulator.
-add_subdirectory(timestamp)
+#add_subdirectory(timestamp)
```

Comment:
Is this change expected?

Reply:
But if we don't annotate the timestamp module, it happen compile error:
**cmake -DCMAKE_INSTALL_PREFIX=./wiredtiger_release -DCMAKE_BUILD_TYPE:STRING=Release ..**
/home/mlg/wiredtiger/test/simulator/timestamp/call_log_manager/call_log_manager.cpp: In member function ‘void call_log_manager::call_log_begin_transaction(const json&)’:
/home/mlg/wiredtiger/test/simulator/timestamp/call_log_manager/call_log_manager.cpp:106:9: error: unused variable ‘ret_expected’ [-Werror=unused-variable]
     int ret_expected = call_log_entry["return"]["return_val"].get<int>();
         ^~~~~~~~~~~~
/home/mlg/wiredtiger/test/simulator/timestamp/call_log_manager/call_log_manager.cpp: In member function ‘void call_log_manager::call_log_commit_transaction(const json&)’:
/home/mlg/wiredtiger/test/simulator/timestamp/call_log_manager/call_log_manager.cpp:145:9: error: unused variable ‘ret_expected’ [-Werror=unused-variable]
     int ret_expected = call_log_entry["return"]["return_val"].get<int>();
         ^~~~~~~~~~~~
[ 39%] Building C object CMakeFiles/wt_objs.dir/src/os_posix/os_setvbuf.c.o
[ 39%] Building C object CMakeFiles/wt_objs.dir/src/os_posix/os_sleep.c.o
/home/mlg/wiredtiger/test/simulator/timestamp/call_log_manager/call_log_manager.cpp: In member function ‘void call_log_manager::call_log_prepare_transaction(const json&)’:
/home/mlg/wiredtiger/test/simulator/timestamp/call_log_manager/call_log_manager.cpp:190:9: error: unused variable ‘ret_expected’ [-Werror=unused-variable]
     int ret_expected = call_log_entry["return"]["return_val"].get<int>();
         ^~~~~~~~~~~~
/home/mlg/wiredtiger/test/simulator/timestamp/call_log_manager/call_log_manager.cpp: In member function ‘void call_log_manager::call_log_query_timestamp(const json&)’:
/home/mlg/wiredtiger/test/simulator/timestamp/call_log_manager/call_log_manager.cpp:238:9: error: unused variable ‘ret_expected’ [-Werror=unused-variable]
     int ret_expected = call_log_entry["return"]["return_val"].get<int>();

Reply:
if we use Debug version, it is right.

Reply:
@Siddhartha8899, is that an issue you are aware of?

Reply:
This error may occur if compiled in release mode as the assert statements that use this variable will be removed. I have created WT-10944.